flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/JobMasterPartitionTrackerImplTest.java

```diff
@@ -110,7 +113,8 @@ public void testReleaseCallsWithLocalResources() {
                         resultPartitionId2, true));
 
         {
-            partitionTracker.stopTrackingAndReleasePartitionsFor(taskExecutorId1);
+            partitionTracker.stopTrackingAndReleasePartitions(
```

Comment:
Looks to me that we do not need this block at all because it is testing the removed `stopTrackingAndReleasePartitionsFor()`

Reply:
Yes, I also had such concern but was hesitate to remote the existing testing block -- at least current code tests the process of un-tracking partitions one by one.

Besides `testReleaseCallsWithoutLocalResources` also has the similar issue.
Do you insists?  I can remote the block if so.

Reply:
I think we can remove the duplicate blocks in `testReleaseCallsWithLocalResources()` and `testReleaseCallsWithoutLocalResources()`.
If to test process of un-tracking partitions one by one, I think we should extract the block logic into one method.